Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -55,6 +56,103 @@
     'Alert.Device.ExternalIPThreatTypesName'
 ]
 
+ALERT_OUTPUT = [
+    'Varonis.Alert.ID',
+    'Varonis.Alert.Name',
+    'Varonis.Alert.Time',
+    'Varonis.Alert.Severity',
+    'Varonis.Alert.Category',
+    'Varonis.Alert.Country',
+    'Varonis.Alert.State',
+    'Varonis.Alert.Status',
+    'Varonis.Alert.CloseReason',
+    'Varonis.Alert.BlacklistLocation',
+    'Varonis.Alert.AbnormalLocation',
+    'Varonis.Alert.NumOfAlertedEvents',
+    'Varonis.Alert.UserName',
+    'Varonis.Alert.By.SamAccountName',
+    'Varonis.Alert.By.PreivilegedAccountType',
+    'Varonis.Alert.By.HasFollowUpIndicators',
+    'Varonis.Alert.On.ContainsFlaggedData',
+    'Varonis.Alert.On.ContainsSensitiveData',
+    'Varonis.Alert.On.Platform',
+    'Varonis.Alert.On.Asset',
+    'Varonis.Alert.On.FileServerOrDomain',
+    'Varonis.Alert.Device.Name',
+    'Varonis.Alert.Device.ContainMaliciousExternalIP',
+    'Varonis.Alert.Device.IPThreatTypes'
+]
+
+EVENT_COLUMNS = [
+    'Event.ID',
+    'Event.Type.Name',
+    'Event.TimeUTC',
+    'Event.Status.Name',
+    'Event.Description',
+    'Event.Location.CountryName',
+    'Event.Location.SubdivisionName',
+    'Event.Location.BlacklistedLocation',
+    'Event.Operation.Name',
+    'Event.EventBy.Name',
+    'Event.EventBy.Type.Name',
+    'Event.EventBy.AccountType.Name',
+    'Event.EventBy.SamAccountName',
+    'Event.EventBy.Domain.Name',
+    'Event.EventBy.IsDisabled',
+    'Event.EventBy.IsStale',
+    'Event.EventBy.IsLockout',
+    'Event.IP',
+    'Event.Device.IsMaliciousExternalIP',
+    'Event.Device.ExternalIPReputationID',
+    'Event.Device.ExternalIPThreatTypesName',
+    'Event.EventOnObjectName',
+    'Event.EventOnResource.ObjectType.Name',
+    'Event.EventOnResource.Folder.Filer.Platform.Name',
+    'Event.EventOnResource.Folder.IsSensitive',
+    'Event.EventOnResource.Folder.Filer.Name',
+    'Event.EventOnUser.IsDisabled',
+    'Event.EventOnUser.IsLockout',
+    'Event.EventOnUser.SamAccountName',
+    'Event.EventOnUser.AccountType.Name',
+    'Event.Destination.IP',
+    'Event.Destination.DeviceName'
+]
+
+EVENT_OUTPUT = [
+    'Varonis.Event.ID',
```

Comment:
Do we have this field in documentation?

Reply:
no, but we need some key field I guess. What do you think?

Reply:
added # TODO need to clarify spec